src/components/Tokens/TokenTable/NetworkFilter.tsx

```diff
@@ -106,7 +106,7 @@ export default function NetworkFilter() {
       <NetworkFilterOption onClick={toggleMenu} aria-label={`networkFilter`} active={open}>
         <StyledMenuContent>
           <NetworkLabel>
-            <Logo src={circleLogoUrl ?? logoUrl} /> {label}
```

Comment:
should we just kill the circle urls?

Reply:
What else uses circle url?

Reply:
logoUrl is for displaying chain switching options, like a main logo. CircleLogo provides circular image that is more visible laid over a token icon, used in the token table & details. LogoUrls don't look good overlaid on token icons because parts of them are transparent